lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -755,9 +755,10 @@ void IMConstPropPass::visitNode(NodeOp node, FieldRef changedFieldRef) {
 
   // Nodes don't fold if they have interesting names, but they should still
   // propagate values.
-  if (hasDontTouch(node.getResult()) ||
-      (node.getAnnotationsAttr() && !node.getAnnotationsAttr().empty()) ||
-      node.isForceable()) {
+  // This is handled by isDeletableWireOrRegOrNode.
```

Comment:
The resulting comment should be cleaned to not be: 

1. Foo does bar
2. Actually, bar does baz

Suggestion: just delete all lines here, including the comment.